src/NewProject/add_constraints_form.cpp

```diff
@@ -22,6 +23,9 @@ addConstraintsForm::addConstraintsForm(QWidget *parent)
 
   ui->m_ckkBoxCopy->setText(tr("Copy sources into project."));
   ui->m_ckkBoxCopy->setCheckState(Qt::CheckState::Checked);
+  ui->select_defineOrder->setChecked(true);
+  bool define_orderFlag = ui->select_defineOrder->isChecked();
```

Comment:
unused code

Reply:
it's not using yet but when backend will interface in the raptor we have to pass these flags to backend. I left them in that way

Reply:
it should be removed.

Reply:
okay